tests/test_dashboard_external_link_accessibility.py

```
@@ -0,0 +1,93 @@
+  category: "test",
+  context: "app-code",
+  message: "message",
+  remediation: "fix",
+  verification: "verify",
+  references: [{json.dumps(reference)}],
+  owasp: [],
+  cwe: []
+}});
+process.stdout.write(detail.innerHTML);
+"""
+    completed = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)